### ISSUE TITLE
6361750 puts REMIND in inconsistent input data state

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [remindmodel/remind#1092]
tl;dr:
- [ ] don't add calibration data incompatible with existing calibration data to remindmodel/CESparameters
- [ ] don't add commits that break REMIND to an approved merge request
      - #980 was approved (by me) on September 26
      - 6361750 was committed and merged on November 8 (!)
- [ ] act on remindmodel/development_issues#146

-----

```diff
 #### Current input data revision (<mainrevision>.<subrevision>) ####
 cfg$inputRevision <- "6.316"

 #### Current CES parameter and GDX revision (commit hash) ####
-cfg$CESandGDXversion <- "346023c22fb098052539fbbfdda2041b5114d82c"
+cfg$CESandGDXversion <- "b577e6de076abd3c12ca0385002af599db370c2c"
```
https://github.com/remindmodel/remind/commit/6361750fa724eebb656aa5e0f693d33ef405f22a

The commit https://github.com/remindmodel/CESparameters/commit/b577e6de076abd3c12ca0385002af599db370c2c contained new calibrations for `SSP2EU` and `SSP1` based on input data revision 6.316 alright, but the packed file `CESparametersAndGDX_b577e6de076abd3c12ca0385002af599db370c2c.tgz` contains calibration data for all other scenarios from different input data revisions, [breaking at least `SDP` and `SSP5` runs](https://mattermost.pik-potsdam.de/rd3/pl/rsa4x9muyirc5dbt7ayinkjd8c) and putting the model into an unknown state with respect to the input data revision and calibration data.

```
$ echo -e "commit | date | file\n-|-|-"; for f in *.gdx; do git --no-pager log -n 1 --format="%h | %cd | $f" --date=short -- "$f"; done | sort -rk 3,3
```
commit | date | file
-|-|-
b577e6d | 2022-11-08 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2EU-GDP_gdp_SSP2EU-En_gdp_SSP2EU-Kap_debt_limit-Reg_62eff8f7.gdx
b577e6d | 2022-11-08 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP1-GDP_gdp_SSP1-En_gdp_SSP1-Kap_debt_limit-Reg_62eff8f7.gdx
98b6bbf | 2022-10-24 | indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2EU-GDP_gdp_SSP2EU-En_gdp_SSP2EU-Kap_debt_limit-FE_med-Reg_2b1450bc.gdx
1074017 | 2022-09-27 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP5-GDP_gdp_SSP5-En_gdp_SSP5-Kap_debt_limit-Reg_62eff8f7.gdx
1074017 | 2022-09-27 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2EU-GDP_gdp_SSP2EU-En_gdp_SSP2EU-Kap_debt_limit-Reg_2b1450bc.gdx
1074017 | 2022-09-27 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SDP_RC-GDP_gdp_SDP_RC-En_gdp_SDP_RC-Kap_debt_limit-Reg_62eff8f7.gdx
1074017 | 2022-09-27 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SDP_MC-GDP_gdp_SDP_MC-En_gdp_SDP_MC-Kap_debt_limit-Reg_62eff8f7.gdx
1074017 | 2022-09-27 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SDP-GDP_gdp_SDP-En_gdp_SDP-Kap_debt_limit-Reg_62eff8f7.gdx
1074017 | 2022-09-27 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SDP_EI-GDP_gdp_SDP_EI-En_gdp_SDP_EI-Kap_debt_limit-Reg_62eff8f7.gdx
f6c6063 | 2022-07-22 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2EU-GDP_gdp_SSP2EU-En_gdp_SSP2EU-Kap_debt_limit-newElast-Reg_2b1450bc.gdx
c35a11e | 2022-06-22 | indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2EU-GDP_gdp_SSP2EU-En_gdp_SSP2_lowEn-Kap_debt_limit-Reg_62eff8f7.gdx
8e55c54 | 2022-01-05 | indu_fixed_shares-buil_simple-tran_complex-POP_pop_SSP2-GDP_gdp_SSP2-En_gdp_SSP2-Kap_debt_limit-FE_med-Reg_2b1450bc.gdx
